pkg/scheduler/event_handler.go

```diff
@@ -231,6 +302,12 @@ func (s *Scheduler) enqueueAffectedBindings(oldCluster, newCluster *clusterv1alp
 
 		var affinity *policyv1alpha1.ClusterAffinity
 		if placementPtr.ClusterAffinities != nil {
+			if binding.Status.SchedulerObservedGeneration != binding.Generation {
+				// redundancy design: if the status of obtained binding is not the latest,
+				// enqueue the binding directly to prevent cluster update event loss.
```

Comment:
```suggestion
				// Hit here means the binding maybe still in the queue waiting
				// for scheduling or its status has not been synced to the
				// cache. Just enqueue the binding to avoid missing the cluster
				// update event.
```